vulkaninfo/CMakeLists.txt

```diff
@@ -1,5 +1,81 @@
 # CMakeLists.txt file for building Vulkaninfo
 
+set(VULKANINFO_INCLUDE_DIRS
+    ${CMAKE_CURRENT_BINARY_DIR}
+    ${CMAKE_CURRENT_BINARY_DIR/..}
+   )
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    include(FindPkgConfig)
+    option(BUILD_WSI_XCB_SUPPORT "Build XCB WSI support" ON)
+    option(BUILD_WSI_XLIB_SUPPORT "Build Xlib WSI support" ON)
+    option(BUILD_WSI_WAYLAND_SUPPORT "Build Wayland WSI support" ON)
```

Comment:
I'm not sure that vulkaninfo has Wayland or Mir support.  It might be good to check and remove these options from the CMake file to avoid implying here in the CMake file that vulkaninfo does more than it really does.